lib/components/app/responsive-webapp.js

```diff
@@ -280,79 +314,81 @@ class RouterWrapperWithAuth0 extends Component {
-          basename={routerConfig && routerConfig.basename}
-          history={history}>
-          <Switch>
-            <Route
-              exact
-              path={[
-                // App root
-                '/',
-                // Load app with preset lat/lon/zoom and optional router
-                // NOTE: All params will be cast to :id in matchContentToUrl due
-                // to a quirk with react-router.
-                // https://github.com/ReactTraining/react-router/issues/5870#issuecomment-394194338
-                '/@/:latLonZoomRouter',
-                '/start/:latLonZoomRouter',
-                // Route viewer (and route ID).
-                '/route',
-                '/route/:id',
-                // Stop viewer (and stop ID).
-                '/stop',
-                '/stop/:id'
-              ]}
-              render={() => <WebappWithRouter {...this.props} />}
-            />
-            <Route
-              component={FavoritePlaceScreen}
-              path={[`${CREATE_ACCOUNT_PLACES_PATH}/:id`, `${PLACES_PATH}/:id`]}
-            />
-            <Route
-              component={SavedTripScreen}
-              path={`${TRIPS_PATH}/:id`}
-            />
-            <Route exact path={ACCOUNT_PATH}>
-              <RedirectWithQuery to={TRIPS_PATH} />
-            </Route>
-            <Route exact path={CREATE_ACCOUNT_PATH}>
-              <RedirectWithQuery to={CREATE_ACCOUNT_VERIFY_PATH} />
-            </Route>
-            <Route
-              // This route lets new or existing users edit or set up their account.
-              component={UserAccountScreen}
-              path={[`${CREATE_ACCOUNT_PATH}/:step`, ACCOUNT_SETTINGS_PATH]}
-            />
-            <Route
-              component={frame(components.TermsOfService)}
-              path={TERMS_OF_SERVICE_PATH}
-            />
-            <Route
-              component={frame(components.TermsOfStorage)}
-              path={TERMS_OF_STORAGE_PATH}
-            />
-            <Route
-              component={SavedTripList}
-              path={TRIPS_PATH}
-            />
-            <Route
-              // This route is called immediately after login by Auth0
-              // and by the onRedirectCallback function from /lib/util/auth.js.
-              // For new users, it displays the account setup form.
-              // For existing users, it takes the browser back to the itinerary search prior to login.
-              component={AfterSignInScreen}
-              path='/signedin'
-            />
-            <Route
-              component={PrintLayout}
-              path='/print'
-            />
-            {/* For any other route, simply return the web app. */}
-            <Route
-              render={() => <WebappWithRouter {...this.props} />}
-            />
-          </Switch>
-        </ConnectedRouter>
+        <IntlProvider locale={locale} messages={this.state.messages} defaultLocale={defaultLocale}>
```

Comment:
Line too long.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2